### ISSUE TITLE
Issue with colors on Project Facets window

### ISSUE THREAD

**Issue by vamsigangarapu** [AObuchow/Eclipse-Spectrum-Theme#127]
After applying the theme I noticed that the project facets window has white background with white text. So, its impossible to look at the text on the window.

**Comment by AObuchow** [comment]
@vamsigangarapu thanks for reporting :) Could you send a picture of the project facets view? I don’t think I’ve used it before :o 

Will try to get this fixed soon for you :)

**Comment by vamsigangarapu** [comment]
@AObuchow Here is the screen shot.
![gIxb9](https://user-images.githubusercontent.com/24572351/90591127-3fda8280-e1b0-11ea-9134-4051a5045d85.png)

**Comment by AObuchow** [comment]
@vamsigangarapu thank you ! And you’re on Windows 10?

**Comment by vamsigangarapu** [comment]
@AObuchow Yes, I am using Windows 10. Also, I noticed similar behavior on Content Assist when **Use Mixed Fonts and colors for Labels** checkbox is checked in Preferences -> General -> Appearance.

**Comment by AObuchow** [comment]
@vamsigangarapu excuse my ignorance, but I'm actually unfamiliar with the projects facets window. Do you know if this come from some plugin you have installed? (I think I'm missing something).

**Comment by AObuchow** [comment]
> @AObuchow Yes, I am using Windows 10 with Eclipse version 2020-06. Also, I noticed similar behavior on Content Assist when **Use Mixed Fonts and colors for Labels** checkbox is checked in Preferences -> General -> Appearance.

Thanks for the info :) Would you mind making a separate bug for this issue? With a photo ideally. I can test this on my Windows machine when I have a moment.

In both cases, do these bugs not occur with the normal Eclipse dark theme?

**Comment by vamsigangarapu** [comment]
@AObuchow right-click on the project -> properties -> Project Facets

**Comment by AObuchow** [comment]
@vamsigangarapu I believe Project Facets [comes from Eclipse WebTools ](https://datacadamia.com/ide/eclipse/facet). I can try and see if I can reproduce the issue after installing webtools, but know that Eclipse webtools is being replaced with [Eclipse Wild Web Developer](https://github.com/eclipse/wildwebdeveloper).

I'll keep you updated.

**Comment by vamsigangarapu** [comment]
@AObuchow You can see Project Facets on a plain Java project as well. I have a Core Java project without any web components attached to the project. When I right click on the project in the project explorer window and went to **properties** I see **Project Facets** there. Initially it will ask us to convert the project when we hit convert it will show the above (Facets) window.

**Comment by AObuchow** [comment]
@vamsigangarapu thanks for the help :) I think however you have a plugin installed that I don’t have, as I don’t have a project facets option. 

I’m installing JBoss tools right now, it contains many plugins and might have the right one to see the project facets options.

**Comment by AObuchow** [comment]
@vamsigangarapu After installing JBoss tools, I can see the projects facets options. 
I think this is a Windows-specific bug, as on Linux, things seem to look OK:
![image](https://user-images.githubusercontent.com/10300119/90592933-c002e700-e1b4-11ea-9a61-ab17af8f0b6b.png)

I'll try to reproduce it on my Windows machine tomorrow and see if I can come up with a fix

**Comment by vamsigangarapu** [comment]
@AObuchow Thank you for the prompt reply

**Comment by AObuchow** [comment]
@vamsigangarapu it might be related to https://bugs.eclipse.org/bugs/show_bug.cgi?id=563475, but I'm not sure. This has been fixed in Eclipse 4.17 which releases soon enough.

Do you mind posting a photo of the project faces look with the standard dark theme? [Is it similar to this? ](https://bugs.eclipse.org/bugs/attachment.cgi?id=282993)

**Comment by vamsigangarapu** [comment]
@AObuchow This is the Project Facets screen with Eclipse default Dark theme:
<img width="712" alt="eclipse" src="https://user-images.githubusercontent.com/24572351/90594700-078b7200-e1b9-11ea-8c0d-8ece4dc23ff1.PNG">

**Comment by AObuchow** [comment]
So the property page that is misbehaving is coming from `org.eclipse.wst.common.project.facet.ui.internal.FacetsPropertyPage`. Just writing this as a little note to myself... I still need to clone org.eclipse.wst

**Comment by vamsigangarapu** [comment]
Sure, no problem. I have currently switched to the eclipse dark theme till the issue is fixed. So, I am good. Please let me know when it is fixed so that I can switch back to the spectrum theme.
